### PR TITLE
Updates the labor camp consoles to reflect new space law

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

You can now set 1500 point sentences. You can also permanently sentence someone to labor camp. Also just a tiny bit of cleanup in the labor stuff code.

## Why It's Good For The Game

Being able to give out accurate sentences is pretty good I think.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>
yea i tested pinky promise

</details>

## Changelog
:cl:
add: You can now permanently sentence people to labor camp
add: You can now give out 1500 point labor camp sentences
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/prisoner/management.dm [hunk 1]
```diff
@@ -26,7 +26,7 @@
 		if(contained_id)
 			dat += text("<A href='?src=[REF(src)];id=eject'>[contained_id]</A><br>")
 			dat += text("Collected Points: [contained_id.points]. <A href='?src=[REF(src)];id=reset'>Reset.</A><br>")
-			dat += text("Card goal: [contained_id.goal].  <A href='?src=[REF(src)];id=setgoal'>Set </A><br>")
+			dat += text("Card goal: [contained_id.goal].  <A href='?src=[REF(src)];id=setgoal'>Set</A> <A href='?src=[REF(src)];id=setpermanent'>Make permanent</A><br>")
```
Comment (ike709) [inline code/game/machinery/computer/prisoner/management.dm]: Shouldn't `Make Permanent` instead be a ternary that checks if it's already permanent or not and has different text if it's already permanent?

File: code/game/machinery/gulag_item_reclaimer.dm [hunk 2]
```diff
@@ -47,8 +45,8 @@
 
 	var/obj/item/card/id/I = user.get_idcard(TRUE)
 	if(istype(I, /obj/item/card/id/prisoner))
-		var/obj/item/card/id/prisoner/P = I
-		if(P.points >= P.goal)
+		var/obj/item/card/id/prisoner/prisonerID = I
+		if(prisonerID.points >= prisonerID.goal && !prisonerID.permanent)
```
Comment (ike709) [inline code/game/machinery/gulag_item_reclaimer.dm:49]: I feel like this should have feedback reminding them that the ID is permanent.

File: code/game/objects/items/cards_ids.dm [hunk 3]
```diff
@@ -598,6 +598,7 @@ update_label("John Doe", "Clowny")
 	registered_name = "Scum"
 	var/goal = 0 //How far from freedom?
 	var/points = 0
+	var/permanent = FALSE
```
Comment (ike709) [inline code/game/objects/items/cards_ids.dm:601]: If this is true it should add a line to the examine text.

File: code/modules/mining/laborcamp/laborstacker.dm [hunk 4]
```diff
@@ -71,15 +69,17 @@ GLOBAL_LIST(labor_sheet_values)
...
 			can_go_home = TRUE
 			data["status_info"] = "Goal met!"
+		else if(prisonerID.permanent)
+			data["status_info"] = "Your sentence is permanent."
 		else
-			data["status_info"] = "You are [(P.goal - P.points)] points away."
+			data["status_info"] = "You are [(prisonerID.goal - prisonerID.points)] points away."
 	else
-		data["status_info"] = "No Prisoner ID detected."
+		data["status_info"] = "No Prisoner cumID detected."
```
Comment (ike709) [inline code/modules/mining/laborcamp/laborstacker.dm]: Yeah?

File: code/modules/mining/laborcamp/laborstacker.dm [hunk 5]
```diff
@@ -93,6 +93,8 @@ GLOBAL_LIST(labor_sheet_values)
 	if(..())
 		return
 	var/mob/M = usr
+	if(!istype(M))
```
Comment (ike709) [inline code/modules/mining/laborcamp/laborstacker.dm]: Is this not already checking for if they're a mob that can actually interact with a computer? If it isn't then this needs a more robust check, and if it is then I don't see how this is necessary.

**Review by ike709 — CHANGES_REQUESTED** [review]
See comments

File: code/modules/mining/laborcamp/laborstacker.dm (hunk 4 above)
Comment (ivanmixo) [reply code/modules/mining/laborcamp/laborstacker.dm]: oh god lmao

File: code/game/machinery/gulag_item_reclaimer.dm (hunk 2 above)
Comment (ivanmixo) [reply code/game/machinery/gulag_item_reclaimer.dm:49]: Well they shouldn't get to the reclaimer in the first place if their sentence is permanent but I could add a line to the UI, I guess.

File: code/game/machinery/computer/prisoner/management.dm [hunk 6]
```diff
@@ -26,7 +26,7 @@
 		if(contained_id)
 			dat += text("<A href='?src=[REF(src)];id=eject'>[contained_id]</A><br>")
 			dat += text("Collected Points: [contained_id.points]. <A href='?src=[REF(src)];id=reset'>Reset.</A><br>")
-			dat += text("Card goal: [contained_id.goal].  <A href='?src=[REF(src)];id=setgoal'>Set </A><br>")
+			dat += text("Card goal: [contained_id.goal].  <A href='?src=[REF(src)];id=setgoal'>Set</A> <A href='?src=[REF(src)];id=setpermanent'>Make [contained_id.permanent ? "permanent" : "temporary"].</A><br>")
```
Comment (ike709) [inline code/game/machinery/computer/prisoner/management.dm]:
```suggestion
			dat += text("Card goal: [contained_id.goal].  <A href='?src=[REF(src)];id=setgoal'>Set</A> <A href='?src=[REF(src)];id=setpermanent'>Make [contained_id.permanent ? "temporary" : "permanent"].</A><br>")
```

**Review by ike709 — APPROVED** [review]

**Pull request merged**